### ISSUE TITLE
regarding the benchmark run option  

### ISSUE THREAD

**Issue by h2th3k** [eembc/coremark-pro#16]
I would like to know if I run the benchmarks individually and not for the certify mark, do I still need to add `-v0` ? Since from the manual it says
> "A value of 0 indicates performance mode, a value of 1 indicates verification mode, which invokes
result checking."

Thanks!

**Comment by petertorelli** [comment]
Yes, you need to include `-v0` because the default is `-v1`, and that only runs a single iteration (`-i1`) which means you won't get a very accurate score.

**Comment by h2th3k** [comment]
is it okay to run like this `-v0 -i1000 -w1 -c1` ? 
also I wonder is it possible to pin the benchmark at a specific core? for instance 
```c
cpuid = sched_getcpu();
if (cpuid = TARGET_CPU) {
     // benchmark code
}
```
then execute the task to run the benchmarks in parallel?

**Comment by petertorelli** [comment]
We began adding an API to set affinity long ago, but it didn't have enough demand and wasn't merged into a branch. If you look at `al_smp.c` and `mith_lib.c` you will see `HAVE_PTHREAD_SETAFFINITY_NP` and `al_item_setaffinity`. It is up to you to implement that as needed.

**Comment by petertorelli** [comment]
Specifically, the function `pthread_setaffinity_np` is used.

**Comment by h2th3k** [comment]
currently by using `-v0 -i1000 -w1 -c1 option` I thought `-c1` will limit the benchmark be running on one CPU only, seems I was wrong?
